Source/Android/app/src/main/java/org/dolphinemu/dolphinemu/utils/SettingsFile.java

```diff
@@ -234,22 +219,35 @@ private static SettingSection sectionFromLine(String line)
 	 * For a line of text, determines what type of data is being represented, and returns
 	 * a Setting object containing this data.
 	 *
-	 * @param current The section currently being parsed by the consuming method.
-	 * @param line The line of text being parsed.
+	 * @param current  The section currently being parsed by the consuming method.
+	 * @param line     The line of text being parsed.
+	 * @param fileName The name of the ini file the setting is in.
 	 * @return A typed Setting containing the key/value contained in the line.
 	 */
-	private static Setting settingFromLine(SettingSection current, String line)
+	private static Setting settingFromLine(SettingSection current, String line, String fileName)
 	{
 		String[] splitLine = line.split("=");
 
 		String key = splitLine[0].trim();
 		String value = splitLine[1].trim();
 
+		int file = SettingsFile.SETTINGS_DOLPHIN;
+
+		switch (fileName)
```

Comment:
The default case would be SETTINGS_DOLPHIN, because it's set to that first and changed if necessary. I can add it to the switch statement though, if you'd like.